content/en/docs/ops/diagnostic-tools/istioctl-analyze/test.sh

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e  # Exit on failure
+set -u  # Unset is an error
+# There is no need to echo, output appears in ops_diagnostic-tools_istioctl-analyze_test_debug.txt
+set -o pipefail
+
+# This script doesn't need a control plane initially and will install Istio when needed
+# @setup profile=none
+
+echo '*** istioctl-analyze step 1 ***'
+_verify_elided snip_analyze_all_namespaces "$snip_analyze_all_namespace_sample_response" 
+
+echo '*** istioctl-analyze step 2 ***'
+_verify_contains snip_fix_default_namespace "$snip_fix_default_namespace_out"
```

Comment:
See comment in md file.
```suggestion
snip_fix_default_namespace
_verify_same try_with_fixed_namespace "$try_with_fixed_namespace_out"
```